module-utils/unicode/utf8/test/unittest_utf8.cpp

```diff
@@ -356,6 +356,13 @@ TEST_CASE("UTF8: Not ASCII combination")
         REQUIRE_FALSE(combination.toASCII().has_value());
     }
 
+    SECTION("Carier info number - 3 digits")
```

Comment:
Could you please add more UTs? For example with other 3/2/1 digits combinations.

Reply:
Working on it.

Reply:
Ok, I see added test cases, thanks.